frontend/src/metabase/nav/containers/MainNavbar/BookmarkList/BookmarkList.tsx

```diff
@@ -30,7 +30,7 @@ const mapDispatchToProps = {
 
 interface CollectionSidebarBookmarksProps {
   bookmarks: BookmarksType;
-  selectedItem?: SelectedEntityItem;
+  selectedItem?: SelectedItem;
```

Comment:
Why change it to `SelectedItem`? Only entities can be bookmarked, so it should expect `SelectedEntityItem`

Reply:
My bad, I tried something here.